controlplane/pkg/plugins/utils.go

```diff
@@ -38,6 +39,32 @@ func StartPlugin(name, registry string, services map[plugins.PluginCapability]in
 		return err
 	}
 
+	go func() {
+		registered := true
+		for ; ; time.Sleep(5 * time.Second) {
```

Comment:
Please move `5 * time.Second` to constant

Reply:
can we also use time.After()?

Reply:
also, I guess we can create env variable for plugins to add possible edit the check period.

Reply:
- [ ] move 5 * time.Second to constant

Reply:
> also, I guess we can create env variable for plugins to add possible edit the check period.

I (and Ilya) have doubts that the option would ever be used

Reply:
@alex-yust , @lobkovilya can we use this env variable for testing? We could set specific time for other tests and set for this test short period. What do you think?

Reply:
@denis-tingajkin do you mean env variable for 'timeout' (like fastTimeout) or for 'check period' (5 second constant here in the code)?

Reply:
@alex-yust oops, sorry, I meant period.

Reply:
- [ ]  use time.After()